apps/core/app/[locale]/(default)/product/[slug]/_components/breadcrumbs.tsx

```diff
@@ -26,9 +26,9 @@ export const BreadCrumbs = async ({ productId }: Props) => {
                 'font-semibold': !isLast,
                 'font-extrabold': isLast,
               })}
-              key={breadcrumb.name}
+              key={name}
             >
-              <Link href="#">{breadcrumb.name}</Link>
+              <Link href={path ?? '#'}>{name}</Link>
```

Comment:
Not sure if we want to do something more complex here if `path` is null, I'm not sure how much of an edge case that is.

Reply:
Maybe it doesn't need to be a link if there is no path?

Reply:
I experimented with this, will also involve remove the `hover:text-primary` styling above, and I started to wonder if it's worth the complexity (I'm honestly not sure why a category would not have a path)

Reply:
>I'm honestly not sure why a category would not have a path

Good question, maybe that is something we need to look into at the API level 🤔

Reply:
Confirmed this is an extreme edge case, don't think we need to account for it any more gracefully than this.